sputnikdao-factory2/src/lib.rs

```diff
@@ -162,30 +173,18 @@ pub extern "C" fn store() {
 #[cfg(test)]
 mod tests {
     use near_sdk::test_utils::{accounts, VMContextBuilder};
-    use near_sdk::{testing_env, PromiseResult, VMContext};
+    use near_sdk::{testing_env, PromiseResult};
 
     use super::*;
 
-    pub fn add_contract(
-        context: &mut VMContext,
-        contract: &mut SputnikDAOFactory,
-    ) -> Base58CryptoHash {
-        context.input = include_bytes!("../../sputnikdao2/res/sputnikdao2.wasm").to_vec();
-        let hash = slice_to_hash(&env::sha256(&context.input));
-        testing_env!(context.clone());
-        contract.factory_manager.store_contract();
-        context.input = vec![];
-        hash
-    }
-
     #[test]
     fn test_basics() {
         let mut context = VMContextBuilder::new();
-        testing_env!(context.current_account_id(accounts(0)).build());
+        testing_env!(context
+            .current_account_id(accounts(0))
+            .predecessor_account_id(accounts(0))
+            .build());
         let mut factory = SputnikDAOFactory::new();
-        let hash = add_contract(&mut context.build(), &mut factory);
```

Comment:
Seems like we should eventually write Sandbox tests for this, as I imagine unit tests aren't nearly powerful enough to test updates